test/passing/tests/js_source.ml.ref

```diff
@@ -9865,3 +9865,6 @@ let xxxxxx =
              __________ *) () = yyyyyyyy in
   { zzzzzzzzzzzzz }
 ;;
+
+module M = struct end
+(* comment long enough to force line splitting *) [@ocaml.warning ""]
```

Comment:
We expect a newline between the comment and the attribute. I feel this should be solved with boxes and breaks but I couldn't come up with a fix.